gatsby-theme-carbon/src/components/AnchorLinks/anchor-links.scss

```diff
@@ -0,0 +1,67 @@
+.#{$prefix}--anchor-links .markdown-ul {
+  width: auto;
+  margin-bottom: 0;
+  margin-top: $spacing-05;
+
+  @include carbon--breakpoint("md") {
+    margin-bottom: $spacing-07;
+  }
+}
+
+.#{$prefix}--anchor-links .markdown-ul li {
```

Comment:
This is a good example of where sass nesting can fail us. If the user shadows the Ul component, the AnchorLink component will break since the `markdown-ul` won't exist.

We should use a new className for the li over nesting as we can't depend on the classNames of any other component existing outside of the current component.